ipaserver/install/server/replicainstall.py

```diff
@@ -918,47 +918,55 @@ def install(installer):
 
 
 def ensure_enrolled(installer):
-    config = installer._config
+    # Prepare options for the installer script
+    args = [paths.IPA_CLIENT_INSTALL, "--no-ntp"]
```

Comment:
This changes the default behaviour. Previously, when a user provided all the mandatory arguments, the installation finished successfully in unattended mode. Now the user gets prompted for confirmations even if he provides all the mandatory attributes.

I think replica should only run in interactive mode if some mandatory attribute is missing to keep backwards compatibility.